src/index.html

```diff
@@ -7,6 +7,8 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
```

Comment:
You might want to get rid of this, we never want to use the roboto font.  Also double check and make sure the font in the component is set to opensans, and is not using roboto or arial .. this has been a problem in the past.